### PR TITLE
[OBSDEF-8704] telemetry report chart changes

### PR DESCRIPTION
**Pull request by vgupta-mickey**
added platform, watch_ns, and schedule as container argument to telem…etry reports

## Purpose
[OBSDEF-8704](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-8704)
_List the changes for this PR_

## PR checklist
- [ ] make test passed
- [ ] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: objectscale-manager/templates/objectscale-manager-app-configmap.yaml [hunk 1]
```diff
@@ -48,24 +48,65 @@ data:
...
           disabled: {{ default false .Values.healthChecks.inventory.disabled}}
+          args:
+            - -platform
+            - {{ .Values.global.platform}}
+            - -watch_ns
+{{- if eq .Values.global.watchAllNamespaces true}}
+            - "" 
+{{- else }}
+            - metadata.namespace
+{{- end }}
```
Comment (eddiepavkovic) [inline objectscale-manager/templates/objectscale-manager-app-configmap.yaml:59]: can we make this simpler.  if the argument is not there then watch all namespaces.
```suggestion
{{- if neq .Values.global.watchAllNamespaces true }}
            - -watch_ns
            - metadata.namespace
{{- end}}
```

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]
can we make it simpler:
if `--watch_ns` is set then watch a namespace only.  if not then watch all.

```
--watch_ns "dellemc-objectscale-domain-c10" 
```

File: objectscale-manager/templates/objectscale-manager-app-configmap.yaml (hunk 1 above)
Comment (vgupta-mickey) [reply objectscale-manager/templates/objectscale-manager-app-configmap.yaml:59]: just to keep this consistent with other watch namespace. We will provide the args with either empty string or it's own namespace.

**Review by eddiepavkovic — APPROVED** [review]
its fine.  it still looks odd to pass in an empty string to watch all namespaces.  hope it works ok in k8s.

**Comment by eddiepavkovic** [comment]
Please fill in the testing section.  there's no problem with passing in an empty string to `--watch_ns ""` ?

**Review by youminhan — APPROVED** [review]

**Pull request merged**